cypress/e2e/faq/faq.js

```diff
@@ -0,0 +1,84 @@
+describe("Checking information on every category faq", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/faq");
```

Comment:
```suggestion
    cy.visit("/faq");
```

You don't need to use the complete base URL as it has been configured at `cypress.json` file.

Reply:
Oh, I see, I didn't look at file cypress.json.